codec-memcache/src/main/java/io/netty/handler/codec/memcache/binary/DefaultBinaryMemcacheRequest.java

```diff
@@ -92,4 +92,14 @@ public BinaryMemcacheRequest touch(Object hint) {
         super.touch(hint);
         return this;
     }
+
+    /**
+     * Copies special metadata hold by this instance to the provided instance
+     *
+     * @param dst The instance where to copy the metadata of this instance to
+     */
+    void copyMeta(DefaultBinaryMemcacheRequest dst) {
```

Comment:
missing `@Override` ?

Reply:
no the signature of the methods are different:
```
super: void copyMeta(AbstractBinaryMemcacheMessage dst)
this : void copyMeta(DefaultBinaryMemcacheRequest dst)
```